### PR TITLE
Assign specific hex code to each member

### LINKED ISSUE #27: Incorporate color coding of background color of the chore cards for each member!
Maybe we can assign a specific color for each user or each chore and store the hex code in the BE so when we fetch the chores we can pull that code and use it for a more cohesive feel... Extension could be adding a color picker to the chore form to allow the user to decide the color!

### PR DESCRIPTION
**Pull request by Kanderson58**
# Description

- Color codes each member on ChoreCard
- Limits number of members to 10

## Type Of Change

- [ ]  Bug fix
- [x]  New feature
- [ ]  Refactoring
- [ ]  Testing

# How Has This Been Tested?

- [x]  local host
- [ ]  dev tools
- [ ]  cypress

# Related Issues:

- [x]  closes  #27 

# Checklist:

- [x]  My code follows the Turing's guidelines of this project
- [x]  I have performed a self-review of my own code
- [x]  No console error messages

### CODE REVIEW

File: src/Components/ChoreCard/ChoreCard.js [hunk 1]
```diff
@@ -1,8 +1,8 @@
 import './ChoreCard.css';
 
-const ChoreCard = ({member, chore, duration}) => {
+const ChoreCard = ({member, allMembers, chore, duration}) => {
   return (
-    <div className='chore-card' tabIndex="0" style={{'backgroundColor':'#3b425d'}}>
+    <div className='chore-card' tabIndex="0" style={{'backgroundColor':allMembers.find(singleMember => singleMember.name === member).color}}>
```
Comment (qrispi) [inline src/Components/ChoreCard/ChoreCard.js:5]: This is really cool. Didn't know you could do that!

File: src/Components/Dashboard/Dashboard.js [hunk 2]
```diff
@@ -111,6 +119,7 @@ const Dashboard = ({ email }) => {
     
   return (
     <div className="dashboard">
+      {console.log(members)}
```
Comment (qrispi) [inline src/Components/Dashboard/Dashboard.js:122]: Forgot a console log!

File: src/Components/HouseForm/HouseForm.js [hunk 3]
```diff
@@ -155,7 +155,7 @@ const HouseForm = ({ email }) => {
           </form>
         )}
         {memberInputs}
-        {!editMember && (
+        {!editMember && members.length < 10 && (
```
Comment (qrispi) [inline src/Components/HouseForm/HouseForm.js:158]: Love this.

**Review by qrispi — APPROVED** [review]
LGTM other than the console.log!

File: src/Components/ChoreCard/ChoreCard.js (hunk 1 above)
Comment (LeftyLincoln) [reply src/Components/ChoreCard/ChoreCard.js:5]: Super cool, well done

File: src/Components/Dashboard/Dashboard.js (hunk 2 above)
Comment (LeftyLincoln) [reply src/Components/Dashboard/Dashboard.js:122]: Love setting the amount of colors to the amount of available chore-doers

**Pull request closed without merging**